pkg/graphql/engine_config_v2.go

```diff
@@ -0,0 +1,142 @@
+package graphql
+
+import (
+	"net/http"
+
+	"github.com/jensneuse/graphql-go-tools/pkg/ast"
+	graphqlDataSource "github.com/jensneuse/graphql-go-tools/pkg/engine/datasource/graphql_datasource"
+	"github.com/jensneuse/graphql-go-tools/pkg/engine/plan"
+)
+
+type EngineV2Configuration struct {
+	schema                   *Schema
+	plannerConfig            plan.Configuration
+	websocketBeforeStartHook WebsocketBeforeStartHook
+}
+
+func NewEngineV2Configuration(schema *Schema) EngineV2Configuration {
+	return EngineV2Configuration{
+		schema: schema,
+		plannerConfig: plan.Configuration{
+			DefaultFlushInterval: 0,
```

Comment:
We should have a global with a reasonable default flush interval.
E.g. 1 second.